EaseUI/EMUIKit/ViewController/EaseConversationListViewController.m

```diff
@@ -172,39 +172,42 @@ - (void)tableViewDidTriggerHeaderRefresh
                                }
                            }];
         dispatch_async(dispatch_get_main_queue(), ^{
-            [weakself.dataArray removeAllObjects];
-            for (EMConversation *converstion in sorted) {
-                EaseConversationModel *model = nil;
-                if (weakself.dataSource && [weakself.dataSource respondsToSelector:@selector(conversationListViewController:modelForConversation:)]) {
-                    model = [weakself.dataSource conversationListViewController:weakself
-                                                       modelForConversation:converstion];
-                }
-                else{
-                    model = [[EaseConversationModel alloc] initWithConversation:converstion];
-                }
-                
-                if (model) {
-                    [weakself.dataArray addObject:model];
+            EaseConversationListViewController *strongSelf = weakself;
```

Comment:
won't this be kinda odd, pointing to weakself but called strongself? why not just use self? 
self.dataSource, self.tableView, etc.